docusaurus/docs/extensions/api/tabs.md

```diff
@@ -23,23 +23,31 @@ _Arguments_
 
 | Key | Type | Description |
 |---|---|---|
-|`TabLocation.RESOURCE_DETAIL`| String | Location for a Tab on a Resource Detail page |
 |`TabLocation.RESOURCE_SHOW_CONFIGURATION`| String | Location for a Tab on a Resource Show Configuration *(From Rancher version v2.14.0)* |
+|`TabLocation.RESOURCE_CREATE_PAGE`| String | Location for a Tab on a Resource Create page  |
+|`TabLocation.RESOURCE_EDIT_PAGE`| String | Location for a Tab on a Resource Edit page  |
```

Comment:
ALL RESOURCE_CREATE_PAGE, RESOURCE_EDIT_PAGE and RESOURCE_DETAIL_PAGE are also from v2.14.0